### ISSUE TITLE
a better design

### ISSUE THREAD

**Issue by graciano** [graciano/tanana#13]
the font type and buttons are just too ugly.

**Comment by graciano** [comment]
@felipescb said to me he could take a look on this :)

**Comment by victorhugorch** [comment]
Maybe I can take a look. What do you think about adding the library [Photon](http://photonkit.com/)?

It was created to build Electron applications similar of a desktop app. It looks interesting if it's the purpose of the tanãnã.

**Comment by graciano** [comment]
@victorhugorch I think maybe one day I could build a web version of tananã. If this doesn't prevent it from happening, go ahead.

**Comment by victorhugorch** [comment]
No, it works perfectly on the web as it shows in the documentation but its appearance is from a desktop application. I'm also cant tell if the library is consolidated and receives frequent updates. 

Another css framework I'm using is [Bulma](http://bulma.io/) but it has plenty of web style. He's the current favorite of the front-end devs.

**Comment by graciano** [comment]
@victorhugorch I liked both. The decision is yours, but keep in mind the mobile first paradigm. Is really annoying when an app like that doesn't resize the window well
